src/test/scala/io/iohk/ethereum/consensus/ConsensusSpec.scala

```diff
@@ -99,7 +111,39 @@ class BlockImportSpec extends AnyFlatSpec with Matchers with ScalaFutures {
 
     (blockQueue.removeSubtree _).expects(*)
 
-    whenReady(blockImport.importBlock(block).runToFuture)(_ shouldBe a[BlockImportFailed])
+    whenReady(consensus.evaluateBranchBlock(block).runToFuture)(
+      _ shouldBe BlockImportFailed("MPTError(io.iohk.ethereum.mpt.MerklePatriciaTrie$MPTException: Invalid Node)")
+    )
+  }
+
+  it should "handle no best block available error when importing to top" in new ImportBlockTestSetup {
```

Comment:
Did you write these two from scratch?

Reply:
I used the existing tests as a blue print